packages/nestjs/README.md

```diff
@@ -56,6 +56,35 @@ export class ExampleService {
 }
 ```
 
+## SentryCron
+
+Use the @SentryCron() decorator to augment the native NestJS `@Cron` decorator to
+send check-ins to Sentry before and after each cron job run.
+
+```typescript
```

Comment:
I think we generally don't use typescript in our docs so that people can copy paste snippets regardless of their setup.

Reply:
Thanks for the hint

Reply:
I think TS is fine for NestJS since it defaults to typescript (similarly to Angular). 

For more context: We sometimes show both for frameworks where there's no clear default  (e.g. [SvelteKit](https://docs.sentry.io/platforms/javascript/guides/sveltekit/manual-setup/)) but, yes, generally rather stick to JS for them.

Reply:
Ah, sorry @nicohrubec and thanks @Lms24